### PR TITLE
[new release] aws-s3 (3 packages) (4.8.1)

### PR DESCRIPTION
**Pull request by andersfugmann**
Ocaml library for accessing Amazon S3

- Project page: <a href="https://github.com/andersfugmann/aws-s3">https://github.com/andersfugmann/aws-s3</a>
- Documentation: <a href="https://andersfugmann.github.io/aws-s3/">https://andersfugmann.github.io/aws-s3/</a>

##### CHANGES:

- [x]: Forward compatibility with Lwt release 5.7/6.0
- [x]: Fix bug causing a leak in filedescriptors (thanks @quernd)

### CODE REVIEW

**Comment by andersfugmann** [comment]
@mseri This PR fixes sha's for the release and updates #25497.

**Comment by mseri** [comment]
@andersfugmann the original submission sources are still available in opam caches, you can download them for example from

https://opam.robur.coop/cache/sha256/26/2658ea176ba9fec42f885d9a8f0d2e652ec945945cf9dc5c40807ebafed09464

Can you replace the release tarball for 4.8.1 on the repository using the old sources linked above and update the URLs instead?

File: packages/aws-s3-async/aws-s3-async.4.8.1/opam [hunk 1]
```diff
@@ -37,8 +37,8 @@ url {
   src:
     "https://github.com/andersfugmann/aws-s3/releases/download/4.8.1/aws-s3-4.8.1.tbz"
   checksum: [
-    "sha256=2658ea176ba9fec42f885d9a8f0d2e652ec945945cf9dc5c40807ebafed09464"
-    "sha512=7e646905202e9f0569bdd95b0e14e89636a69b6c43f9c14ef2268d6f8d565b89bcfe303e1a56b27d6c5ca39ebceb4bb543315f34432eb2096b304e18a7221b83"
+    "sha256=58ae9305c24364abce2b0f14046454b0efcec87e08859154f1aae2f7d8661115"
```
Comment (mseri) [inline packages/aws-s3-async/aws-s3-async.4.8.1/opam:40]: The old sources and the new sources are not the same. In addition to the opam file changes and the github action changes, the aws-s3-async differs at line 90 by
```diff
- | Some ms -> Some (Async.after (Core.Time_float.Span.of_ms (float ms)))
+ | Some ms -> Some (Async.after (Time_float_unix.Span.of_int_ms ms))
```

**Comment by andersfugmann** [comment]
Thanks @mseri. I've updated the artifact so everything should work again as expected. Thanks for the help!

Closing this PR, as the issue has been resolved.

**Pull request closed without merging**